torch/utils/benchmark/utils/timer.py

```diff
@@ -21,6 +22,57 @@ def timer() -> float:
     timer = timeit.default_timer
 
 
+PYTHON = "Python"
+CPP = "C++"
```

Comment:
Insisting that these be capitalized is not very PyTorch-y (where we say "cuda" not "CUDA")

Reply:
Done.